src/main/java/com/dev/cinema/dao/impl/TicketDaoImp.java

```diff
@@ -0,0 +1,35 @@
+package com.dev.cinema.dao.impl;
+
+import com.dev.cinema.dao.TicketDao;
+import com.dev.cinema.lib.Dao;
+import com.dev.cinema.model.Ticket;
+import com.dev.cinema.util.HibernateUtil;
+import exceptions.DataProcessingException;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+
+@Dao
+public class TicketDaoImp implements TicketDao {
+    @Override
+    public Ticket add(Ticket ticket) {
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            session.persist(ticket);
```

Comment:
let's be consistent and try to use the same methods for the same actions, if there is no particular reason to use a different method